activesupport/lib/active_support/notifications.rb

```diff
@@ -231,6 +231,13 @@ def instrument(name, payload = {})
       #   ActiveSupport::Notifications.subscribe(/render/) do |event|
       #     @event = event
       #   end
+      #
+      # Only accepts a String, Regex or empty(matches all events) event name.
```

Comment:
I don't think we need to repeat this again since the comments above already states 

>      # You can subscribe to events by passing a String to match exact event
>      # names, or by passing a Regexp to match all events that match a pattern.